src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/FeedbackAssessment/FeedbackAssessment.utils.test.ts

```diff
@@ -0,0 +1,116 @@
+import { formatToNumberArray, hasAnswerValue, getUpdatedValues } from './FeedbackAssessment.utils';
+
+describe('formatToNumberArray', () => {
+  test.each`
+    stringArray             | expected        | description
+    ${[]}                   | ${[]}           | ${'should return an empty array when the input array is empty'}
+    ${['1', '2', '3', '4']} | ${[1, 2, 3, 4]} | ${'should convert an array of strings to an array of numbers'}
+  `('$description', ({ stringArray, expected }) => {
+    const result = formatToNumberArray(stringArray);
+    expect(result).toEqual(expected);
+  });
+});
+
+describe('hasAnswerValue', () => {
+  test.each`
+    value                 | expected | description
+    ${''}                 | ${false} | ${'should return false for an empty string'}
+    ${[]}                 | ${false} | ${'should return false for an empty array'}
+    ${'Hello World!'}     | ${true}  | ${'should return true for a non-empty string'}
+    ${12}                 | ${true}  | ${'should return true for a non-empty number'}
+    ${['item1', 'item2']} | ${true}  | ${'should return true for a non-empty array'}
+  `('$description', ({ value, expected }) => {
+    const result = hasAnswerValue(value);
+    expect(result).toBe(expected);
+  });
+});
+
+describe('getUpdatedValues', () => {
+  it('should return values for a multiple selection item when answers are the same(skipped) and has not been edited before', () => {
+    const defaultValues = [{ itemId: 'item1', answers: [], edited: null }];
+    const currentItem = { itemId: 'item1', answers: [], edited: null };
+    const prevItemIds: string[] = [];
+    const updatedItemIds: string[] = [];
+
+    const result = getUpdatedValues(defaultValues, currentItem, prevItemIds, updatedItemIds);
+
+    expect(result).toEqual({ edited: null, itemIds: ['item1'] });
+  });
+
+  it('should return values for a multiple selection item when answers are the same(skipped) and has been edited before', () => {
```

Comment:
maybe it's better to use test everywhere in project instead of it? for consistence